ProcessLib/BoundaryCondition/GenericNaturalBoundaryCondition-impl.h

```diff
@@ -45,6 +39,23 @@ GenericNaturalBoundaryCondition<BoundaryConditionData,
             dof_table_bulk.getNumberOfVariableComponents(variable_id));
     }
 
+    if (_bc_mesh.getDimension() + 1 != global_dim)
```

Comment:
How about the case of a 3D mesh getting 1D BC?

Reply:
With out "ordinary" natural BCs a 1D BC for a 3D domain doesn't make sense, does it? For something like line sources the case is different, of course. But they don't fit our natural BCs anyway.

Reply:
I see that BCs on lines for 3D problem are excluded from this type of definition.